src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj

```diff
@@ -21,9 +21,9 @@
 
 <!-- Microsoft.Build.Locator is only used when debugging, and the compiler will skip copying this dependency from Release assemblies we insert because we only refer to it conditionally with the DEBUG configuration.
   Uncomment the following when debugging. Also uncomment the MSBuildLocator code from Program.cs -->
-  <!-- <ItemGroup>
+   <!--<ItemGroup>
```

Comment:
No, we don't need that change. I made such an effort to avoid commiting this file (and uncommenting the related lines in program.cs) on each commit, but all it took was one slip-up ☹️  Then, instead of "reverting" the file, I tried using the IDE to comment out the same lines, but it didn't do it the same way as the code before.

I also prefer the choose-when for debug mode. I don't know what problem was trying to be solved by having debug and release builds work the same, but in my experience working on this bug, it makes development harder. The debug-only code was just easier to work with and had no impact on the optimized binaries that we ship to customers.